spec/models/solidus_bolt/bolt_configuration_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'spec_helper'
+
+RSpec.describe SolidusBolt::BoltConfiguration, type: :model do
+  let(:column_list) {
+    [
+      'id',
+      'bearer_token',
+      'environment_url',
+      'merchant_public_id',
+      'merchant_id',
+      'api_key',
+      'signing_secret',
+      'publishable_key',
+      'created_at',
+      'updated_at'
+    ]
+  }
+
+  describe 'Check Model Integrity' do
+    it 'has correct table name' do
+      expect(described_class.table_name).to eq 'solidus_bolt_bolt_configurations'
+    end
+
+    it 'has correct attributes' do
+      expect(described_class.new.attributes.keys.sort).to eq column_list.sort
+    end
+  end
```

Comment:
Are you sure about these tests? 
We don't have any tests like this. 
It looks to me these tests are checking something that should be already tested on ActiveRecord. I'd prefer if you'll remove them.

Reply:
removed these tests
check here [365a6d4](https://github.com/nebulab/solidus_bolt/commit/365a6d43e7e8a049df560bde89c04183e14bda7a)